x-pack/legacy/plugins/integrations_manager/public/screens/detail/screenshots.tsx

```diff
@@ -19,15 +19,19 @@ export function Screenshots(props: ScreenshotProps) {
   const { images } = props;
 
   // for now, just get first image
-  const src = toImage(images[0].src);
+  const image = images[0];
+  const hasCaption: boolean = images[0].title ? true : false;
 
   const horizontalPadding: number = parseInt(theme.eui.paddingSizes.xl, 10) * 2;
-  const bottomPadding: number = parseInt(theme.eui.paddingSizes.xl, 10) * 1.75;
+  const fullVerticalPadding: number = parseInt(theme.eui.paddingSizes.xl, 10) * 1.75;
+  const padding = hasCaption
+    ? `${theme.eui.paddingSizes.xl} ${horizontalPadding}px ${fullVerticalPadding}px`
```

Comment:
Not critical, which is why I didn't say anything initially, but I struggle with the 3 value version of padding/margin. I can parse what a 2 and 4 value statement means, but 3 always makes me want to look up the spec.

I'd appreciate changing to the 4 value format, but not a blocker.

Reply:
just think of it as going clockwise and when you run out for the last one (left side padding), it's the same as the right. :)